package.json

```diff
@@ -75,10 +75,10 @@
     "patternfly": "^3.42.0",
     "patternfly-react": "^2.3.3",
     "prop-types": "^15.6.0",
-    "react": "^16.2.0",
+    "react": "^16.3.1",
```

Comment:
This also allows react `16.4.0` which has been released. Should I package that or did you intend `~16.3.1~?

Reply:
I intended 16.3.1, but I don't mind to package 16.4.0

Reply:
If it stays `^16.3.1` then I'll package 16.4.0 since that's what Travis will also install while testing.